static/js/constants.js

```diff
@@ -24,6 +24,16 @@ export const EDUCATION_LEVELS = [
   {value: DOCTORATE, label: "Doctorate"}
 ];
 
+export const FILTER_NAMES = {
```

Comment:
maybe `SEARCH_FACET_FIELD_LABEL_MAP` since the value is being used more as a 'label' than a title. either one would work. i'll leave it to you